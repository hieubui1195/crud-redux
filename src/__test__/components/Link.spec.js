import React from "react";
import Link from "../../components/Link";
import renderer from "react-test-renderer";

it("reders correctly", () => {
  const tree = renderer.create(<Link>Facebook</Link>).toJSON();
  expect(tree).toMatchSnapshot();
});

it("rederers correctly inline snapshots", () => {
  const tree = renderer.create(<Link>Inline</Link>).toJSON();
  expect(tree).toMatchInlineSnapshot(`
<a
  className="normal"
  href="#"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  Inline
</a>
`);
});
