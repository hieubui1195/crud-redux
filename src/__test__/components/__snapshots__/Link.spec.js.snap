// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reders correctly 1`] = `
<a
  className="normal"
  href="#"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  Facebook
</a>
`;
